### PR TITLE
added new test and confirm account info is stored in database

### PR DESCRIPTION
**Pull request by maggiee0606**

### CODE REVIEW

File: src/test/java/utilities/DatabaseUtils.java [hunk 1]
```diff
@@ -35,9 +35,9 @@ public static ResultSet executeQuery(String query) {
         //2. Execute the query - We use this whenever we want to execute the query
         // We have created
         try {
-            rs = stmt.executeQuery("Select * from employees");
-        } catch (SQLException e) {
-            throw new RuntimeException(e);
+            rs = stmt.executeQuery(query);
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (charlieedaviss) [inline src/test/java/utilities/DatabaseUtils.java:40]: Nice Catch!

File: src/test/resources/features/NewSavings.feature [hunk 2]
```diff
@@ -9,3 +9,8 @@ Feature: Create New Savings
     Then user opens new savings page
```
Comment (charlieedaviss) [inline src/test/resources/features/NewSavings.feature:9]: Maggie is the REAL mvp

File: src/test/java/utilities/DatabaseUtils.java (hunk 1 above)
Comment (maggiee0606) [reply src/test/java/utilities/DatabaseUtils.java:40]: Thank you, working with PROs here

**Comment by TetianaHusar** [comment]
Great job, Meggie.

**Pull request merged**